### PR TITLE
Add minimal execution based test to existing notebook.

### PR DESCRIPTION
**Pull request by sorensenjs**
(Note that this test actually downloads glove embedding + the civil comments data.)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,102 @@
+absl-py==0.11.0
```
Comment (idk3) [inline requirements.txt:1]: These requirements are quite constraining -- at a future point (TF2.4?) we should be able to relax == to >= and remove some of them

File: run_tests.sh [hunk 2]
```diff
@@ -5,3 +5,7 @@ set -x
 pushd unintended_ml_bias
 python model_bias_analysis_test.py
 popd
+
+# IPython "Tests" execute notebooks
+jupyter-nbconvert --to notebook --execute \
```
Comment (idk3) [inline run_tests.sh:10]: very nice :)

**Review by idk3 — APPROVED** [review]

File: presentations/FAT_Star_Tutorial_Measuring_Unintended_Bias_in_Text_Classification_Models_with_Real_Data.ipynb [hunk 3]
```diff
@@ -371,7 +371,7 @@
         "    return pad_sequences(tokenizer.texts_to_sequences(texts), maxlen=MAX_SEQUENCE_LENGTH)\n",
         "\n",
         "# Load the first model from disk.\n",
-        "model = load_model('model_2_3_4.h5')"
+        "model = load_model(b'model_2_3_4.h5')"
```
[the remark this answers is not in the document]
Comment (sorensenjs) [reply presentations/FAT_Star_Tutorial_Measuring_Unintended_Bias_in_Text_Classification_Models_with_Real_Data.ipynb]: Reverted

File: run_tests.sh (hunk 2 above)
Comment (sorensenjs) [reply run_tests.sh:10]: well...

File: requirements.txt (hunk 1 above)
Comment (sorensenjs) [reply requirements.txt:1]: Best practices here seem hard to find.

**Pull request merged**